xray/test/test_plot.py

```diff
@@ -657,7 +707,7 @@ def test_imshow_called(self):
         self.assertTrue(self.imshow_called(self.darray.plot.imshow))
 
     def test_xy_pixel_centered(self):
-        self.darray.plot.imshow()
+        self.darray.plot.imshow(yincrease=None)
```

Comment:
good catch.  At one point this test was failing for me so I was playing around, trying to understand why.  Forgot to roll back that change.  Fixed in d793e3a.